src/private/privateAPIs.ts

```diff
@@ -31,6 +34,14 @@ export function getUserJoinedTeams(
 ): void {
   ensureInitialized();
 
```

Comment:
I see this API was introduced after we introduced versioning change in Teams SDK. Then, why is there no versioning check in this API ? You are just placing a check now and that too for android. What about iOS ?

Reply:
Not sure why the API didn't have the versioning parameter as a part of it's signature, maybe @sachingupta can help with that. As highlighted in the document attached in the description. iOS doesn't have the mentioned issue and hence doesn't need any special checks in place

Reply:
If the user is on ios and the ios app is old and doesn't have implementation of this API the web app developers will not get an error message(OLD_PLATFORM). They will simply find the API no-op. To ensure passing proper error message back to the user Pankaj had introduced versioning. It will be good to keep it consistent with all APIs.

Reply:
That is correct. I completely understand the rationale behind the previously introduced versioning strategy and the repercussions of not having a version check in the implementation of the API. However I am not the primary contributor for the API, and the API has already been implemented without a version check, with a method signature that is not compatible with the versioning strategy.

The scope of this PR is to only address the issues faced on android, the existing behaviour on any of the other clients is not to be taken up in this PR. Making the API compliant with the versioning strategy requires a larger discussion and should be taken up as a separate item. 
@ydogandjiev  @sachingupta to chime in on how to take this on further